internal/commands/viewport.go

```diff
@@ -52,6 +52,17 @@ func updateViewport(msg tea.Msg, m model) (tea.Model, tea.Cmd) {
 			}
 			cmds = append(cmds, m.UpdateList())
 
+		case key.Matches(msg, ViewportKeyMap.Read):
+			current, err := m.commands.store.GetItemByID(*m.selectedArticle)
```

Comment:
Flow wise it might make sense to ignore on `AutoRead` since it'll negate what it did ::

```suggestion
			if m.commands.config.AutoRead {
				return m, nil
			}
			current, err := m.commands.store.GetItemByID(*m.selectedArticle)
```

Reply:
Agreed